roles/ceph-rgw/tasks/pre_requisite.yml

```diff
@@ -41,6 +41,12 @@
     group: "{{ key_group }}"
   when: cephx
 
+- name: trigger creation of ceph-radosgw systemd unit file
+  command: chkconfig --add ceph-radosgw
```

Comment:
I'm fine with running this command all the time, I just want to make sure that if we run the command multiple times it will always report 0 is success and/or already exist :).
The conditional check works for me too.